app/graphql/mutations/integrations/sync_invoice.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Mutations
+  module Integrations
+    class SyncInvoice < BaseMutation
+      include AuthenticableApiUser
+      include RequiredOrganization
+
+      REQUIRED_PERMISSION = 'organization:integrations:update'
+
+      graphql_name 'SyncIntegrationInvoice'
+      description 'Sync integration invoice'
+
+      argument :invoice_id, ID, required: true
+
+      type Types::IntegrationItems::Object.collection_type, null: false
+
+      def resolve(**args)
+        ::Integrations::Aggregator::Invoices::CreateJob.perform_later(invoice:)
+
```

Comment:
I guess, we should first find invoice by `invoice_id`. Here, we don't have invoice object at all